### PR TITLE
Flavor resource class

### PR DESCRIPTION
**Pull request by JohnGarbutt**

### CODE REVIEW

File: etc/p3-config/p3-config.yml [hunk 1]
```diff
@@ -329,34 +329,44 @@ p3_flavor_compute_A:
   ram: 131072
   disk: 371
   vcpus: 64
+  extra_specs:
+    "resources:CUSTOM_COMPUTE_A": 1
```
Comment (markgoddard) [inline etc/p3-config/p3-config.yml:333]: Shouldn't this be CUSTOM_CUSTOM_COMPUTE_A? :p

File: etc/p3-config/p3-config.yml [hunk 2]
```diff
@@ -367,6 +377,8 @@ p3_flavor_compute_A_magnum:
   disk: 271
   vcpus: 64
   ephemeral: 100
+  extra_specs:
+    "resources:CUSTOM_COMPUTE_A": 1
```
Comment (markgoddard) [inline etc/p3-config/p3-config.yml:381]: Will the ephemeral disk still be passed through?

**Review by markgoddard — APPROVED** [review]

File: etc/p3-config/p3-config.yml (hunk 1 above)
Comment (JohnGarbutt) [reply etc/p3-config/p3-config.yml:333]: No :p

File: etc/p3-config/p3-config.yml (hunk 2 above)
Comment (markgoddard) [reply etc/p3-config/p3-config.yml:381]: I can't remember if you followed up about this.

Comment (JohnGarbutt) [reply etc/p3-config/p3-config.yml:381]: so it gets passed through in the same way, I think its all in that patch call you just changed.

**Comment by markgoddard** [comment]
Ok, let's merge then.

**Pull request merged**